src/main/java/com/theironyard/saveyourprogress/Main.java

```diff
@@ -0,0 +1,32 @@
+package com.theironyard.saveyourprogress;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Created by melmo on 12/7/16.
+ */
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        Scanner input = new Scanner(System.in);
+
+        try {
+            Car car = Car.loadCar();
```

Comment:
Avoid putting more code in the `try` than what is needed.   Every line in this try could potentially exception and since `Exception` is a super class of them all it would be caught.  However you don't actually check to see what the error is.